devices/HumanControlBoard/HumanControlBoard.h

```diff
@@ -130,6 +132,12 @@ class hde::devices::HumanControlBoard final
     bool getTorqueRanges(double* min, double* max) override { return false; }
     bool getMotorTorqueParams(int j, yarp::dev::MotorTorqueParameters* params) override { return false; }
     bool setMotorTorqueParams(int j, const yarp::dev::MotorTorqueParameters params) override { return false; }
+
+    // IControlLimits
+    bool setLimits(int axis, double min, double max) override  { return false; }
```

Comment:
@lrapetti May be a warning message here to indicate setting of limits is not allowed will be indicative

Reply:
Currently we have no error message for all the `set` methods but we just return `false`. Do you think we should add this kind of message for all the methods?

Reply:
Don't bother with it for this PR